src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -67,18 +67,20 @@ type DataCache = Record<string, Promise<LoadDataReturnValue>>;
 // V3 dictionary helper functions
 function updateExpiredDepositsV3(dict: ExpiredDepositsToRefundV3, deposit: V3DepositWithBlock): void {
   // A deposit refund for a deposit is invalid if the depositor has a bytes32 address input for an EVM chain. It is valid otherwise.
-  if (chainIsEvm(deposit.originChainId) && !isValidEvmAddress(deposit.depositor)) {
+  if (chainIsEvm(deposit.originChainId) && !deposit.depositor.isValidEvmAddress()) {
     return;
   }
-  const { originChainId, inputToken } = deposit;
+  const { originChainId, inputToken: _inputToken } = deposit;
+  const inputToken = _inputToken.toString();
```

Comment:
@dohaki @gsteenkamp89 Can you think of any way we can make the toString automatic, so we can rely on it happening in the background when `inputToken` is supplied as a key to an Object? Not sure if that's even possible, but if not, this usage scenario represents a sharp edge that we'll probably run into.